config_defaults.yml

```diff
@@ -28,12 +28,14 @@
 Gemfile:
   required:
     ':development, :unit_tests':
-      - gem: json
       - gem: metadata-json-lint
       - gem: puppet_facts
       # newer version required for ruby 1.9 compat
       - gem: puppet-blacksmith
         version: '>= 3.4.0'
+      # older version required for ruby 1.9 compat, as it is pulled in as dependency of puppet, this has to be carried by the module
+      - gem: json_pure
+        version: '<= 2.0.1'
```

Comment:
I take it you don't yet have conditionals? 

We ended up dropping this as high up as Puppet, since it wasn't in any group - see https://github.com/puppetlabs/modulesync_configs/pull/78/files